app/src/main/java/com/github/se/cyrcle/ui/addParking/attributes/AttributesPicker.kt

```diff
@@ -96,19 +102,19 @@ fun AttributesPicker(
   val topBoxHeight = screenHeight * 0.10f // 10% of screen height for top box
   val bottomBoxHeight = screenHeight * 0.15f // 15% of screen height for bottom box
 
-  val title = remember { mutableStateOf("") }
-
-  val description = remember { mutableStateOf("") }
-
-  val rackType = remember { mutableStateOf<ParkingAttribute>(ParkingRackType.TWO_TIER) }
-  val capacity = remember { mutableStateOf<ParkingAttribute>(ParkingCapacity.XSMALL) }
-  val protection = remember { mutableStateOf<ParkingAttribute>(ParkingProtection.INDOOR) }
-  val hasSecurity = remember { mutableStateOf(false) }
+  val title = rememberSaveable { mutableStateOf("") }
+  val description = rememberSaveable { mutableStateOf("") }
+  val rackType = rememberSaveable { mutableStateOf<ParkingAttribute>(ParkingRackType.TWO_TIER) }
+  val capacity = rememberSaveable { mutableStateOf<ParkingAttribute>(ParkingCapacity.XSMALL) }
+  val protection = rememberSaveable { mutableStateOf<ParkingAttribute>(ParkingProtection.NONE) }
+  val hasSecurity = rememberSaveable { mutableStateOf(false) }
 
   val location = mapViewModel.selectedLocation.collectAsState().value!!
   LaunchedEffect(location) { addressViewModel.search(location.center) }
   val suggestedAddress = addressViewModel.address.collectAsState().value
-  LaunchedEffect(suggestedAddress) { title.value = suggestedAddress.displayRelevantFields() }
+  LaunchedEffect(suggestedAddress) {
+    if (title.value.isEmpty()) title.value = suggestedAddress.displayRelevantFields()
+  }
 
   fun onSubmit() {
```

Comment:
There is a missing documentation here as well

Reply:
what do you mean missing documentation? If you're talking about the onSubmit,it's just defined here to be used as a lambda. It's basically a variable. In any case, I did not make this code and am therefore not responsible for its documentation

Reply:
Ohhh i didn't see that you weren't the one who wrote this code my bad.